collection-scripts/gather_nodes

```diff
@@ -47,6 +47,26 @@ do
         fi
     done
 
+    oc exec -it $pod -n node-gather -- [ -d /host/opt/cni/bin ]
+    if [[ $? -eq 0 ]]; then
+        CNI_BIN_PATH="${NODES_PATH}/$node/opt/cni/bin"
+        mkdir -p ${CNI_BIN_PATH}
+        for i in $(oc exec -it $pod -n node-gather -- ls /host/opt/cni/bin)
+        do
+            oc exec -it $pod -n node-gather -- cat /host/opt/cni/bin/$i > "$CNI_BIN_PATH/$i"
+        done
+    fi
+
+    oc exec -it $pod -n node-gather -- [ -d /host/etc/cni/net.d ]
+    if [[ $? -eq 0 ]]; then
+        CNI_COFIG_PATH=${NODES_PATH}/$node/etc/cni/net.d
```

Comment:
you reuse $NODE_PATH which is defined on line 33 instead of ${NODES_PATH}/$node

Reply:
Done